client/my-sites/media-library/test/list-item-image.jsx

```diff
@@ -1,22 +1,18 @@
 /**
  * @jest-environment jsdom
  */
-import { shallow } from 'enzyme';
+import { render, screen } from '@testing-library/react';
 import resize from 'calypso/lib/resize-image-url';
 import ListItemImage from 'calypso/my-sites/media-library/list-item-image';
 import fixtures from './fixtures';
 
 const WIDTH = 450;
 
-describe( 'MediaLibraryListItem image', () => {
-	let wrapper;
-
-	beforeEach( () => {
-		if ( wrapper ) {
-			wrapper.unmount();
-		}
-	} );
+jest.mock( 'calypso/my-sites/media-library/media-image', () => {
+	return ( props ) => <div data-testid={ props.src }>{ props.children }</div>;
```

Comment:
Let's use destructuring here, too:

```suggestion
	return ( { children, src } ) => <div data-testid={ src }>{ children }</div>;
```